### PR TITLE
HCloud CSI driver v1.1.5

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: lib/pharos/resources/hcloud/hcloud-csi.yml.erb [hunk 1]
```diff
@@ -135,7 +135,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: hcloud-csi-driver
-          image: kontenapharos/hcloud-csi-driver:1.1.4-kontena.1
+          image: hetznercloud/hcloud-csi-driver:1.1.5
```
Comment (jnummelin) [inline lib/pharos/resources/hcloud/hcloud-csi.yml.erb]: Should we "prefix" this with `docker.io/...`?

Comment (jakolehm) [reply lib/pharos/resources/hcloud/hcloud-csi.yml.erb]: Does not hurt, fixed.

**Comment by nevalla** [comment]
There are some other changes that we need to test out before merge: https://github.com/hetznercloud/csi-driver/pull/43

**Comment by jakolehm** [comment]
Seems to work. I'll open a separate issue about manifest update.

**Pull request merged**